cmd/packaging/windows-msi.go

```diff
@@ -120,7 +194,7 @@ var WindowsMsiTask = &packagingTask{
 	},
 }
 
-func windowsMsiProcessFiles(path string) {
+func windowsMsiProcessFiles(path, fileSeparator string) {
```

Comment:
Instead of adding `fileSeparator` as argument, can we make this a local variable in the function? It doesn't seem that the value given to this argument is ever anything other than `filepath.Separator`.

Reply:
Also here, `pathSeparator` would be a better name I think.

Reply:
Hm I looked up the reference of `filepath.Separator` which points to `os.PathSeparator` which is simply a `/`. I'll reverted the path separator, because it has no effect on the code or it's execution.

Reply:
No you should use filepath.Separator! The os.PathSeparator is defined in a OS-specific Go file., e.g. path_unix.go or path_windows.go. So it's `\` on windows.